NOM_visualizations/python/nom_data.ipynb

```diff
@@ -0,0 +1,3353 @@
+{
```

Comment:
<p>Also, maybe explain why are grabbing the <code>has_input</code>.Its nice to set the reader up for understanding your thought process behind traversing the schema. E.g. The <code>has_input</code> will be important for grabbing the raw data objects that are the input into the Nom Analysis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>